test/index.spec.ts

```diff
@@ -85,6 +85,8 @@ describe(`\nTest strategy "${strategy}"`, () => {
   });
 
   it('Returned addresses should be either same case as input addresses or checksum addresses', () => {
+
+
     expect(
```

Comment:
Please remove these new lines
```suggestion
    expect(
```